### PR TITLE
Refonte UX FAQ

### PR DESCRIPTION
**Pull request by sandrica89**
Notes: 
- Search bar was removed because behavior is unclear and technical implementation undefined (to be added in a second version if still needed)
- Use accordion instead of modal for questions

Remaining tasks:
- [x] Fix sidemenu overlap between tabs **(need react-dsfr update)**
- [x] Fix tab width issue when adding "Je suis un médecin" tab content
- [x] Fix display when clicking on question accordion at the end of the page (answer is not visible)  **(need react-dsfr update)**
- [x] Add flyers (in FaqProcess component?)
- [x] Fix responsive 
- [x] Update link from psy reimbursement page to faq
- [x] Add cypress tests (to test tab and sidemenu behavior on questions filtering)

Optional tasks for react-dsfr:
* ~Add "active" property in SideMenuLink~
* "title" property from SideMenu shouldn't be required according to dsfr (it displays an error in console)

### CODE REVIEW

File: frontend/src/components/Faq/FaqTab.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                    }
+            >
+              {section.title}
+            </SideMenuLink>
+          ))}
+        </SideMenu>
+        <Col n="md-8 sm-12" className={styles.sections}>
+          {items[type].sections.map((section, index) => (
+            <div ref={refs[index]} key={section.name} id={section.id ? section.id : null}>
+              <Title as="h2" look="h4">{section.title}</Title>
```
Comment (desoindx) [inline frontend/src/components/Faq/FaqTab.jsx:58]: pas d'interet a le forcer en h2 ? 
```suggestion
              <Title as="h4" look="h4">{section.title}</Title>
```

File: frontend/src/App.css [hunk 2]
```diff
@@ -96,3 +96,14 @@
...
+.fr-tabs {
+  overflow: unset;
+}
+
+/* overwrite DSFR default */
+@media (min-width: 48em) {
+  .fr-tabs__panel {
+    padding: 0; 
+  }
+}
```
Comment (desoindx) [inline frontend/src/App.css:109]: EOF missing

File: frontend/src/components/Faq/faqProcess.cssmodule.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.label {
+  color: var(--bf500);
+  font-weight: bold;
+  text-align: center;
+}
+
+.link {
+  width: max-content;
+  margin: 4px 8px;
+}
```
Comment (desoindx) [inline frontend/src/components/Faq/faqProcess.cssmodule.scss:10]: EOF missing

File: frontend/src/components/Faq/faqTab.cssmodule.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.sections > div:not(:first-of-type) {
+  margin-top: 24px;
+}
+
+.menu {
+  margin-bottom: 24px;
+}
```
Comment (desoindx) [inline frontend/src/components/Faq/faqTab.cssmodule.scss:7]: EOF missing

File: frontend/src/components/PsyListing/PsyListing.jsx [hunk 5]
```diff
@@ -56,6 +56,7 @@ const PsyListing = () => {
       description={`Il y a actuellement ${psychologists.length} partenaires du dispositif d‘accompagnement.
       La liste est mise à jour quotidiennement, revenez la consulter si vous n‘avez pas pu trouver de psychologue.`}
       background="yellow"
+      dataTestId="psyListPage"
```
Comment (desoindx) [inline frontend/src/components/PsyListing/PsyListing.jsx:59]: data-test-id to be coherent ?

Comment (desoindx) [reply frontend/src/components/PsyListing/PsyListing.jsx:59]:
```suggestion
      data-test-id="psyListPage"
```

File: frontend/src/components/PsyListing/PublicPsychologistProfile.jsx [hunk 6]
```diff
@@ -66,6 +66,7 @@ const PublicPsychologistProfile = () => {
       title={psychologist ? `${psychologist.lastName.toUpperCase()} ${camelize(psychologist.firstNames)}` : 'Loading'}
       description={psychologist ? psychologist.description : ''}
       background="yellow"
+      dataTestId="publicPsyProfilePage"
```
Comment (desoindx) [inline frontend/src/components/PsyListing/PublicPsychologistProfile.jsx:69]: data-test-id to be coherent
```suggestion
      data-test-id="publicPsyProfilePage"
```

**Review by desoindx — CHANGES_REQUESTED** [review]

File: frontend/src/components/Faq/FaqTab.jsx (hunk 1 above)
Comment (sandrica89) [reply frontend/src/components/Faq/FaqTab.jsx:58]: Il me semble que c'est une règle liée à l'accessibilité (je ne l'ai pas trouvé dans le référentiel RGAA cela dit :thinking:), d'avoir un ordre de titre qui se suit h1, h2, etc. Je pense que c'est pour cette raison qu'il y a `as` et `look`.

File: frontend/src/components/PsyListing/PsyListing.jsx (hunk 5 above)
Comment (sandrica89) [reply frontend/src/components/PsyListing/PsyListing.jsx:59]: If I rename it `data-test-id` I won't be able to call the prop in Page component like this :confused: 
It's done the same way for `DateInput`.

**Pull request merged**